src/main/java/nl/avans/drivio/model/FuelCar.java

```diff
@@ -1,15 +1,62 @@
 package nl.avans.drivio.model;
 
+import org.springframework.stereotype.Component;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table
 public class FuelCar extends Car{
-    double literPer100Km;
-    int tankSize;
-    int range;
 
-    public FuelCar(int carId, String brand, String model, String fuelType, int buildYear, String numberPlate, String carType, String gearBox, User owner, User renter, double literPer100Km, int tankSize, int range) {
-        super(carId, brand, model, fuelType, buildYear, numberPlate, carType, gearBox, owner, renter);
+    private double literPer100Km;
+    private int tankSize;
+    @Column(name = "RangeCar")

```

Comment:
Zullen we dezelfde naam als in de database aanhouden? Dus in de klasse ook RangeCar of CarRange ervan maken?

Mening?

Reply:
Mee eens, ik ga dit vanavond even aanpassen. Daarna push ik het en kan de code gemerged worden

Reply:
Ik heb de wijziging uitgevoerd (zie commit [[Feat: Changed the requested changes from PR#25](https://github.com/leonvsc/Drivio/pull/25/commits/8fa43389fecb9ea25438afd652b9d161e75421ff))](https://github.com/leonvsc/Drivio/pull/25/commits/8fa43389fecb9ea25438afd652b9d161e75421ff)